pkg/build/admission/defaults/admission.go

```diff
@@ -71,6 +71,28 @@ func (b BuildDefaults) applyPodDefaults(pod *kapi.Pod) {
 	for k, v := range b.config.Annotations {
 		addDefaultAnnotations(k, v, pod.Annotations)
 	}
+
+	// Apply default resources
+	defaultResources := b.config.Resources
+	if len(pod.Spec.Containers[0].Resources.Limits) == 0 {
```

Comment:
just double-checking - is it guaranteed that Containers[0] exists / is any logic required if len(Containers) > 1?